src/transaction/log_page_buffer.c

```diff
@@ -8854,6 +8863,39 @@ logpb_restore (THREAD_ENTRY * thread_p, const char *db_fullname, const char *log
       try_level = (FILEIO_BACKUP_LEVEL) (try_level - 1);
     }
 
+  // Incremental backup volumes often do not include volume header pages.
+  // Accordingly, it is necessary to set unlinked volumes after all volume header pages are restored.
+  for (int i = 0; i < unlinked_volinfo.size (); i++)
```

Comment:
It's recommended for a STL container to use for-range instead of accessing with an index.
for vector. This line may generate a warning.
`for (const auto& volinfo :  unlinked_volinfo)` // volinfo := tuple<int,string,string>
or if you accept to use map I recommended above
`for (const auto &[volid, vol_names]:unlinked_volinfo)` //vol_names = pair<string, string>